Source/CardStackController.m

```diff
@@ -188,43 +188,21 @@ - (void)cardTitlePanDidStart:(CardView *)card {
 }
 
 - (void)card:(CardView *)card titlePannedByDelta:(CGPoint)delta {
-    self.maximumTotalTitleBarHeightBeforeSearchAppears = 0;
-    for (NSUInteger index = 0; index < self.currentCardIndex; index++) {
-        CardView *card = [self.cards objectAtIndex:index];
-        self.maximumTotalTitleBarHeightBeforeSearchAppears += (card.titleBarHeight * card.scale - CardStackOffsetToAvoidAreaBelowTheTitleToBecomeVisible) * CardStackMaximumVisibleTitleBarProportion;
-    }
-
-    if ((card.tag != self.currentCardIndex &&
-        card.tag != self.cards.count - 1) ||
-        self.cards.count == 1) {
+    [self calculateMaximumTotalTitleBarHeightBeforeSearchAppears];
+    if ([self thisIsNotTheCorrectCard:card]) {
```

Comment:
I see your point. How about this?

```BOOL panningCurrentCard = (card.tag == self.currentCardIndex);
BOOL panningLastCard = (card.tag == self.cards.count - 1);
BOOL isOnlyOneCard = (self.cards.count == 1);
if (!panningCurrentCard || !panningLastCard || isOnlyOneCard) {```

Probably it describes the intention the best?